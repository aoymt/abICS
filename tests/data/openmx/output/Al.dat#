System.CurrrentDirectory ./
System.Name Al
level.of.stdout 1
level.of.fileout 1
Species.Number 1
<Definition.of.Atomic.Species
 Al Al7.0-s3p3d2 Al_PBE19
Definition.of.Atomic.Species>
Atoms.Number 2
Atoms.SpeciesAndCoordinates.Unit FRAC
<Atoms.SpeciesAndCoordinates
   1   Al    0.00000000000000    0.00000000000000    0.00000000000000     1.5     1.5
   2   Al    0.50000000000000    0.50000000000000    0.50000000000000     1.5     1.5
Atoms.SpeciesAndCoordinates>
scf.XcType GGA-PBE
scf.SpinPolarization off
scf.ElectronicTemperature 300.0
scf.energycutoff 120.0
scf.maxIter 100
scf.EigenvalueSolver cluster
scf.Kgrid 1 1 1
scf.Mixing.Type rmm-diis
scf.Init.Mixing.Weight 0.200
scf.Min.Mixing.Weight 0.001
scf.Max.Mixing.Weight 0.200
scf.Mixing.History 7
scf.Mixing.StartPulay 4
scf.criterion 1.0e-10
scf.lapack.dste dstevx
MD.Type nomd
MD.maxIter 1
MD.TimeStep 1.0
MD.Opt.criterion 1.0e-4
DATA.PATH /Users/yomichi/Dropbox/source/github.com/issp-center-dev/abICS/tests/data/openmx/bin/../DFT_DATA19
Atoms.UnitVectors.Unit  Ang
<Atoms.UnitVectors
  4.000000000000000   0.000000000000000   0.000000000000000 
  0.000000000000000   4.000000000000000   0.000000000000000 
  0.000000000000000   0.000000000000000   4.000000000000000 
Atoms.UnitVectors>


scf.restart    on


scf.fixed.grid     0.00000000000000    0.00000000000000    0.00000000000000


MD.Current.Iter   1


<Atoms.Unitvectors.Velocity
  0.0000000000000000000000   0.0000000000000000000000   0.0000000000000000000000 
  0.0000000000000000000000   0.0000000000000000000000   0.0000000000000000000000 
  0.0000000000000000000000   0.0000000000000000000000   0.0000000000000000000000 
Atoms.Unitvectors.Velocity>


<NPT.WV.F0
  0.0000000000000000000000   0.0000000000000000000000   0.0000000000000000000000 
  0.0000000000000000000000   0.0000000000000000000000   0.0000000000000000000000 
  0.0000000000000000000000   0.0000000000000000000000   0.0000000000000000000000 
NPT.WV.F0>
