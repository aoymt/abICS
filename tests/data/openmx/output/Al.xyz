2 

Al    0.00000   0.00000   0.00000  -0.000000000232674  -0.000000000044903   0.000000000302157
Al    2.00000   2.00000   2.00000   0.000000000187093   0.000000000098149  -0.000000000293812
