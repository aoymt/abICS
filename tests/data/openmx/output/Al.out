***********************************************************
***********************************************************

  This calculation was performed by OpenMX Ver. 3.9
  using 1 MPI processes and 1 OpenMP threads.

  Wed Mar  4 22:16:19 2020

***********************************************************
***********************************************************

System.CurrrentDirectory ./
System.Name Al
level.of.stdout 1
level.of.fileout 1
Species.Number 1
<Definition.of.Atomic.Species
 Al Al7.0-s3p3d2 Al_PBE19
Definition.of.Atomic.Species>
Atoms.Number 2
Atoms.SpeciesAndCoordinates.Unit FRAC
<Atoms.SpeciesAndCoordinates
 1 Al 0.0 0.0 0.0 1.5 1.5
 2 Al 0.5 0.5 0.5 1.5 1.5
Atoms.SpeciesAndCoordinates>
scf.XcType GGA-PBE
scf.SpinPolarization off
scf.ElectronicTemperature 300.0
scf.energycutoff 120.0
scf.maxIter 100
scf.EigenvalueSolver cluster
scf.Kgrid 1 1 1
scf.Mixing.Type rmm-diis
scf.Init.Mixing.Weight 0.200
scf.Min.Mixing.Weight 0.001
scf.Max.Mixing.Weight 0.200
scf.Mixing.History 7
scf.Mixing.StartPulay 4
scf.criterion 1.0e-10
scf.lapack.dste dstevx
MD.Type nomd
MD.maxIter 1
MD.TimeStep 1.0
MD.Opt.criterion 1.0e-4
DATA.PATH /Users/yomichi/Dropbox/source/github.com/issp-center-dev/abICS/tests/data/openmx/bin/../DFT_DATA19
Atoms.UnitVectors.Unit Ang
<Atoms.UnitVectors
 4.0 0.0 0.0
 0.0 4.0 0.0
 0.0 0.0 4.0
Atoms.UnitVectors>


***********************************************************
***********************************************************

  Required cutoff energy (Ryd) for 3D-grids = 120.0000
      Used cutoff energy (Ryd) for 3D-grids = 125.9243, 125.9243, 125.9243
  Num. of grids of a-, b-, and c-axes = 27, 27, 27

  Num.Grid1.    27
  Num.Grid2.    27
  Num.Grid3.    27


  Cell_Volume =    431.893314374666 (Bohr^3)
  GridVol     =      0.021942453608 (Bohr^3)
  Cell vectors (bohr) of the grid cell (gtv)
    gtv_a =  0.279959405715,  0.000000000000,  0.000000000000
    gtv_b =  0.000000000000,  0.279959405715,  0.000000000000
    gtv_c =  0.000000000000,  0.000000000000,  0.279959405715
    |gtv_a| =  0.279959405715
    |gtv_b| =  0.279959405715
    |gtv_c| =  0.279959405715

***********************************************************
***********************************************************


***********************************************************
***********************************************************
                  SCF history at MD= 1                    
***********************************************************
***********************************************************

   SCF=   1  NormRD=  1.000000000000  Uele= -1.304329809912
   SCF=   2  NormRD=  0.038207413712  Uele= -1.298348051378
   SCF=   3  NormRD=  0.363420974657  Uele= -1.274860188155
   SCF=   4  NormRD=  0.365572818478  Uele= -1.274493317810
   SCF=   5  NormRD=  0.364741275535  Uele= -1.274497334123
   SCF=   6  NormRD=  0.001953495642  Uele= -1.276190147148
   SCF=   7  NormRD=  0.000447002907  Uele= -1.276128803768
   SCF=   8  NormRD=  0.000397888909  Uele= -1.276128950428
   SCF=   9  NormRD=  0.000003513116  Uele= -1.276129030506
   SCF=  10  NormRD=  0.000001787684  Uele= -1.276129100435
   SCF=  11  NormRD=  0.000001613343  Uele= -1.276129100574
   SCF=  12  NormRD=  0.000004949723  Uele= -1.276129112985
   SCF=  13  NormRD=  0.000001950331  Uele= -1.276129112216
   SCF=  14  NormRD=  0.000001965062  Uele= -1.276129112251

*******************************************************
        Total energy (Hartree) at MD = 1        
*******************************************************

  Uele.         -1.276129112251

  Ukin.          1.596075296783
  UH0.         -53.933269186563
  UH1.           0.003030290725
  Una.          -2.071095209326
  Unl.           1.062411413417
  Uxc0.         -1.008778387730
  Uxc1.         -1.008778387730
  Ucore.        50.976714838492
  Uhub.          0.000000000000
  Ucs.           0.000000000000
  Uzs.           0.000000000000
  Uzo.           0.000000000000
  Uef.           0.000000000000
  UvdW.          0.000000000000
  Uch.           0.000000000000
  Utot.         -4.383689331931

  UpV.           0.000000000000
  Enpy.         -4.383689331931
  Note:

  Utot = Ukin+UH0+UH1+Una+Unl+Uxc0+Uxc1+Ucore+Uhub+Ucs+Uzs+Uzo+Uef+UvdW+Uch

  Uele:   band energy
  Ukin:   kinetic energy
  UH0:    electric part of screened Coulomb energy
  UH1:    difference electron-electron Coulomb energy
  Una:    neutral atom potential energy
  Unl:    non-local potential energy
  Uxc0:   exchange-correlation energy for alpha spin
  Uxc1:   exchange-correlation energy for beta spin
  Ucore:  core-core Coulomb energy
  Uhub:   LDA+U energy
  Ucs:    constraint energy for spin orientation
  Uzs:    Zeeman term for spin magnetic moment
  Uzo:    Zeeman term for orbital magnetic moment
  Uef:    electric energy by electric field
  UvdW:   semi-empirical vdW energy
  Uch:    penalty term to create a core hole

  UpV:    pressure times volume
  Enpy:   Enthalpy = Utot + UpV
  (see also PRB 72, 045121(2005) for the energy contributions)



  Chemical potential (Hartree)      -0.095283279229

***********************************************************
***********************************************************
            Eigenvalues (Hartree) for SCF KS-eq.           
***********************************************************
***********************************************************

   Chemical Potential (Hartree) =  -0.09528327922903
   Number of States             =   6.00000000000000
   HOMO =  2
   Eigenvalues
                Up-spin            Down-spin
          1  -0.40916979546114  -0.40916979546114
          2  -0.13426999521230  -0.13426999521230
          3  -0.09462476809577  -0.09462476809577
          4  -0.09462476590762  -0.09462476590762
          5  -0.09462476248568  -0.09462476248568
          6  -0.00177829268366  -0.00177829268366
          7  -0.00177827943125  -0.00177827943125
          8   0.13952064749962   0.13952064749962
          9   0.13952064769148   0.13952064769148
         10   0.13952065176422   0.13952065176422
         11   0.23133175788430   0.23133175788430
         12   0.23133176053161   0.23133176053161
         13   0.23133176303280   0.23133176303280
         14   0.33732787942210   0.33732787942210
         15   0.33732788220671   0.33732788220671
         16   0.42768719125547   0.42768719125547
         17   0.50017062812136   0.50017062812136
         18   0.50017063278372   0.50017063278372
         19   0.50017063410384   0.50017063410384
         20   0.66079853005307   0.66079853005307
         21   0.66079853455367   0.66079853455367
         22   0.66079854449986   0.66079854449986
         23   0.84055070825223   0.84055070825223
         24   1.02197819528923   1.02197819528923
         25   1.02197821524103   1.02197821524103
         26   1.07533173076904   1.07533173076904
         27   1.07533173177761   1.07533173177761
         28   1.07533173399040   1.07533173399040
         29   1.32398313269146   1.32398313269146
         30   1.32398329147186   1.32398329147186
         31   1.48197508801947   1.48197508801947
         32   1.48197509048891   1.48197509048891
         33   1.48197509128664   1.48197509128664
         34   1.62161806160032   1.62161806160032
         35   1.63806048651817   1.63806048651817
         36   1.63806049249211   1.63806049249211
         37   1.63806049605552   1.63806049605552
         38   2.48840993477429   2.48840993477429
         39   3.02394463744894   3.02394463744894
         40   3.02394469420769   3.02394469420769
         41   3.02394470703981   3.02394470703981
         42   3.43516672375005   3.43516672375005
         43   3.43516680654846   3.43516680654846
         44   3.43516691741012   3.43516691741012

***********************************************************
***********************************************************
                   Mulliken populations                    
***********************************************************
***********************************************************

  Total spin moment (muB)   0.000000000

                    Up spin      Down spin     Sum           Diff
      1   Al      1.500360642  1.500360642   3.000721285   0.000000000
      2   Al      1.499639358  1.499639358   2.999278715   0.000000000

 Sum of MulP: up   =     3.00000 down          =     3.00000
              total=     6.00000 ideal(neutral)=     6.00000


  Decomposed Mulliken populations

    1   Al          Up spin      Down spin     Sum           Diff
            multiple
  s           0    1.049013292  1.049013292   2.098026584   0.000000000
   sum over m      1.049013292  1.049013292   2.098026584   0.000000000
  s           1   -0.045717882 -0.045717882  -0.091435764   0.000000000
   sum over m     -0.045717882 -0.045717882  -0.091435764   0.000000000
  s           2   -0.003425027 -0.003425027  -0.006850055   0.000000000
   sum over m     -0.003425027 -0.003425027  -0.006850055   0.000000000
   sum over m+mul  0.999870382  0.999870382   1.999740765   0.000000000
  px          0    0.163178657  0.163178657   0.326357314   0.000000000
  py          0    0.163178985  0.163178985   0.326357971   0.000000000
  pz          0    0.163179139  0.163179139   0.326358279   0.000000000
   sum over m      0.489536782  0.489536782   0.979073563   0.000000000
  px          1    0.001282860  0.001282860   0.002565720   0.000000000
  py          1    0.001282862  0.001282862   0.002565724   0.000000000
  pz          1    0.001282863  0.001282863   0.002565727   0.000000000
   sum over m      0.003848586  0.003848586   0.007697171   0.000000000
  px          2    0.002368294  0.002368294   0.004736588   0.000000000
  py          2    0.002368298  0.002368298   0.004736597   0.000000000
  pz          2    0.002368301  0.002368301   0.004736601   0.000000000
   sum over m      0.007104893  0.007104893   0.014209786   0.000000000
   sum over m+mul  0.500490260  0.500490260   1.000980520   0.000000000
  d3z^2-r^2   0    0.000000000  0.000000000   0.000000000   0.000000000
  dx^2-y^2    0    0.000000000  0.000000000   0.000000000   0.000000000
  dxy         0    0.000000000  0.000000000   0.000000000   0.000000000
  dxz         0    0.000000000  0.000000000   0.000000000   0.000000000
  dyz         0   -0.000000000 -0.000000000  -0.000000000   0.000000000
   sum over m      0.000000000  0.000000000   0.000000000   0.000000000
  d3z^2-r^2   1   -0.000000000 -0.000000000  -0.000000000   0.000000000
  dx^2-y^2    1   -0.000000000 -0.000000000  -0.000000000   0.000000000
  dxy         1   -0.000000000 -0.000000000  -0.000000000   0.000000000
  dxz         1   -0.000000000 -0.000000000  -0.000000000   0.000000000
  dyz         1    0.000000000  0.000000000   0.000000000   0.000000000
   sum over m     -0.000000000 -0.000000000  -0.000000000   0.000000000
   sum over m+mul  0.000000000  0.000000000   0.000000000   0.000000000

    2   Al          Up spin      Down spin     Sum           Diff
            multiple
  s           0    1.049224631  1.049224631   2.098449262   0.000000000
   sum over m      1.049224631  1.049224631   2.098449262   0.000000000
  s           1   -0.045640535 -0.045640535  -0.091281071   0.000000000
   sum over m     -0.045640535 -0.045640535  -0.091281071   0.000000000
  s           2   -0.003454478 -0.003454478  -0.006908955   0.000000000
   sum over m     -0.003454478 -0.003454478  -0.006908955   0.000000000
   sum over m+mul  1.000129618  1.000129618   2.000259236   0.000000000
  px          0    0.162831369  0.162831369   0.325662737   0.000000000
  py          0    0.162831699  0.162831699   0.325663398   0.000000000
  pz          0    0.162831849  0.162831849   0.325663698   0.000000000
   sum over m      0.488494917  0.488494917   0.976989833   0.000000000
  px          1    0.001293700  0.001293700   0.002587399   0.000000000
  py          1    0.001293702  0.001293702   0.002587404   0.000000000
  pz          1    0.001293703  0.001293703   0.002587406   0.000000000
   sum over m      0.003881105  0.003881105   0.007762209   0.000000000
  px          2    0.002377902  0.002377902   0.004755805   0.000000000
  py          2    0.002377907  0.002377907   0.004755814   0.000000000
  pz          2    0.002377909  0.002377909   0.004755818   0.000000000
   sum over m      0.007133718  0.007133718   0.014267437   0.000000000
   sum over m+mul  0.499509740  0.499509740   0.999019480   0.000000000
  d3z^2-r^2   0    0.000000000  0.000000000   0.000000000   0.000000000
  dx^2-y^2    0    0.000000000  0.000000000   0.000000000   0.000000000
  dxy         0    0.000000000  0.000000000   0.000000000   0.000000000
  dxz         0    0.000000000  0.000000000   0.000000000   0.000000000
  dyz         0   -0.000000000 -0.000000000  -0.000000000   0.000000000
   sum over m      0.000000000  0.000000000   0.000000000   0.000000000
  d3z^2-r^2   1    0.000000000  0.000000000   0.000000000   0.000000000
  dx^2-y^2    1   -0.000000000 -0.000000000  -0.000000000   0.000000000
  dxy         1   -0.000000000 -0.000000000  -0.000000000   0.000000000
  dxz         1   -0.000000000 -0.000000000  -0.000000000   0.000000000
  dyz         1    0.000000000  0.000000000   0.000000000   0.000000000
   sum over m     -0.000000000 -0.000000000  -0.000000000   0.000000000
   sum over m+mul  0.000000000  0.000000000   0.000000000   0.000000000

***********************************************************
***********************************************************
                    Dipole moment (Debye)                  
***********************************************************
***********************************************************

 Absolute D       46.43062913

                      Dx                Dy                Dz
 Total            -26.80673578      -26.80673632      -26.80673658
 Core              28.81924079       28.81924079       28.81924079
 Electron         -55.62597657      -55.62597711      -55.62597737
 Back ground       -0.00000000       -0.00000000       -0.00000000

***********************************************************
***********************************************************
       xyz-coordinates (Ang) and forces (Hartree/Bohr)  
***********************************************************
***********************************************************

<coordinates.forces
  2
    1    Al     0.00000   0.00000   0.00000   0.000000000233  0.000000000045 -0.000000000302
    2    Al     2.00000   2.00000   2.00000  -0.000000000187 -0.000000000098  0.000000000294
coordinates.forces>

***********************************************************
***********************************************************
       Fractional coordinates of the final structure       
***********************************************************
***********************************************************

     1     Al     0.00000000000000   0.00000000000000   0.00000000000000
     2     Al     0.50000000000000   0.50000000000000   0.50000000000000

***********************************************************
***********************************************************
               Computational Time (second)                 
***********************************************************
***********************************************************

   Elapsed.Time.       155.291

                               Min_ID   Min_Time       Max_ID   Max_Time
   Total Computational Time =     0      155.291          0      155.291
   readfile                 =     0        8.787          0        8.787
   truncation               =     0        3.140          0        3.140
   MD_pac                   =     0        0.003          0        0.003
   OutData                  =     0        0.530          0        0.530
   DFT                      =     0      142.813          0      142.813

*** In DFT ***

   Set_OLP_Kin              =     0        5.871          0        5.871
   Set_Nonlocal             =     0        3.775          0        3.775
   Set_ProExpn_VNA          =     0       37.780          0       37.780
   Set_Hamiltonian          =     0       15.688          0       15.688
   Poisson                  =     0        0.179          0        0.179
   Diagonalization          =     0        0.350          0        0.350
   Mixing_DM                =     0        0.134          0        0.134
   Force                    =     0       64.353          0       64.353
   Total_Energy             =     0        1.958          0        1.958
   Set_Aden_Grid            =     0        0.054          0        0.054
   Set_Orbitals_Grid        =     0        0.134          0        0.134
   Set_Density_Grid         =     0       12.373          0       12.373
   RestartFileDFT           =     0        0.039          0        0.039
   Mulliken_Charge          =     0        0.027          0        0.027
   FFT(2D)_Density          =     0        0.000          0        0.000
   Others                   =     0        0.099          0        0.099
